File: src/main/java/space/serphantom/project/Time.java
package space.serphantom.project;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class Time {
    public String getTimeData() {
        LocalDateTime now = LocalDateTime.now();
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm:ss dd-MM-yyyy");

        return now.format(formatter);
    }
}

File: src/main/kotlin/space/serphantom/project/Time.kt
package space.serphantom.project

import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

class Time {
    val timeData: String
        get() {
            val now = LocalDateTime.now()
            val formatter = DateTimeFormatter.ofPattern("HH:mm:ss dd-MM-yyyy")

            return now.format(formatter)
        }
}
